Exercises/variables-declaration/variables-declaration.js

```diff
@@ -0,0 +1,26 @@
+// Declare a variable which will be encapsulated from global scope.
+
+const variableGlobal = 'global'; //global
```

Comment:
Each time when you create a new branch for a new task you should create it not from other task branch but from `master`. Then in you PR you will have only changes that are related to your current task.
Currently you created branch `numbers` from `variables-declaration` branch and that's why you have unnecessary changes in this branch.
You can leave this PR for now but in future please base your new branches on master so we can check your code only in files that should be checked.

Reply:
Mike, thank you for review and comment. Let me clarify the steps to skip future inconvenience
1. switch to the master from the current branch
2. create new branch 
3. make commit from the created branch